appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
+  - node_modules
+  - '%APPDATA%\npm-cache'
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
This tell Git to convert CRLF to LF.... why would we need this? We should just ensure that the base code has no CRLF and we could eschew this, right?